src/generators/notebookgenerator/itemrenderer.tsx

```diff
@@ -23,8 +178,16 @@ export function notebookItemRenderer(
       let collapsed = cellRef!.model.metadata.get(
         'toc-hr-collapsed'
       ) as boolean;
-      collapsed = collapsed != undefined ? collapsed : false;
+      collapsed = collapsed !== undefined ? collapsed : false;
       cellRef!.model.metadata.set('toc-hr-collapsed', !collapsed);
+      if (cellRef && tracker) {
```

Comment:
Why the `tracker` check here? Is it allowed to be `null`?

Reply:
Removed.